client/components/tinymce/plugins/wpcom-view/views/embed/index.js

```diff
@@ -1,140 +1,51 @@
-/**
- * External dependencies
- */
-
-import EventEmitter from 'events';
-import { defer, uniqueId } from 'lodash';
 /**
  * Internal dependencies
  */
-import EmbedsListStore from 'lib/embeds/list-store';
-import EmbedsStore from 'lib/embeds/store';
-import actions from 'lib/embeds/actions';
 import EmbedView from './view';
+import getSiteEmbeds from 'state/selectors/get-site-embeds';
 import { getSelectedSiteId } from 'state/ui/selectors';
-import { SELECTED_SITE_SUBSCRIBE, SELECTED_SITE_UNSUBSCRIBE } from 'state/action-types';
```

Comment:
After this PR, the `SELECTED_SITE_SUBSCRIBE` actions can be removes, as their only usage is gone. You'll still need to subscribe to the Redux store, but that shouldn't be done with `store.subscribe()`. Layers of indirection with actions and middleware are not needed.

Reply:
Like the other cleanups, I'm happy to address it in another PR to keep this one as small as possible. It's far from being small already 😉

Reply:
Addressed in #39285.